### PR TITLE
Fix repository standardization issues

### PR DESCRIPTION
**Pull request by rekmarks**
- Add `@lavamoat/allow-scripts`, update readme
- Update ESLint dependencies, config, and scripts

### CODE REVIEW

**Review by brad-decker — APPROVED** [review]

File: .eslintrc.js [hunk 1]
```diff
@@ -1,28 +1,27 @@
...
+  extends: ['@metamask/eslint-config', '@metamask/eslint-config-nodejs'],
+
+  overrides: [
+    {
+      files: ['src/*.js'],
+      parserOptions: {
+        sourceType: 'module',
+      },
+      rules: {
+        camelcase: ['error'],
```
Comment (Gudahtt) [inline .eslintrc.js:21]: :eyes: 

Why is this listed? The default camelcase rule should work fine here.

File: .eslintrc.js [hunk 2]
```diff
@@ -1,28 +1,27 @@
...
+      },
```
Comment (rekmarks) [inline .eslintrc.js:22]:
```suggestion
```

File: .eslintrc.js (hunk 1 above)
Comment (rekmarks) [reply .eslintrc.js]: Doh, I had overrides that I removed, and forgot to remove the rule itself. Fixed in 8683953

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Pull request merged**